package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "react-survey",
+  "version": "0.1.0",
+  "private": true,
```

Comment:
Should we add the [engines](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#engines) versions as well? This way it is easy to find back what node/yarn version is working with this project :) 

Or, if you want to use [asdf](https://github.com/asdf-vm/asdf) (which seems very interesting), then a `.tool-versions` file would be enough ;-)

Reply:
yeah, we can specify the engines versions as well, done!